libs/langchain-community/src/retrievers/tests/amazon_knowledge_bases.int.test.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-process-env */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { test } from "@jest/globals";
+import { AmazonKnowledgeBaseRetriever } from "../amazon_knowledge_bases.js";
+
+test("AmazonKnowledgeBaseRetriever", async () => {
+  const retriever = new AmazonKnowledgeBaseRetriever({
+    topK: 10,
+    knowledgeBaseId: "GZ3S9ZLSIM",
```

Comment:
Can you change to an env var?

Reply:
Changed the knowledgeBaseId and aws region